setup.cfg

```diff
@@ -38,6 +38,7 @@ install_requires =
 
 [options.extras_require]
 tests =
+    openlineage-airflow>=0.19.2
```

Comment:
why do we need to add openlineage to tests? Shouldnt they run without adding this